### DISCUSSION TITLE
Extensions Stop Working After a While.

### DISCUSSION THREAD

**Discussion by AyaanZaveri** [coder/code-server#4092] [Q&A] [answered]
For some random reason, sometimes the all of my extensions stop working after a while. It says activating extensions but my rclone tasks, my live-server, etc, still don't show up. Even after I refresh the page, the extensions don't come back. Is there a reason this is happening, and is there a way to bring it back? 

Thanks!

**Hosted On:** Railway

![ezgif-4-4150fcf6f46d](https://user-images.githubusercontent.com/63752541/131884649-29b57dd8-3438-41f6-8dff-9baab45dfcf7.gif)

**Comment by jsjoeio** [comment, marked as answer]
Hmm...that's very strange. Nothing specific comes to mind. What happens if you rebuild and redeploy your env, does that fix it?

**Reply by AyaanZaveri** [reply]
I'm gonna test that right now.

**Reply by AyaanZaveri** [reply]
Now everything is blank.

**Reply by jsjoeio** [reply]
Hmmm 🤔 I mean it'll be hard to help you debug like this. Can you share the repo with your setup or maybe a video? I wonder if it has to do with rclone or something else

**Reply by AyaanZaveri** [reply]
**Repo**: https://github.com/AyaanZaveri/cs-code-server/
**Dockerfile:** https://github.com/AyaanZaveri/cs-code-server/blob/main/Dockerfile

**Reply by AyaanZaveri** [reply]
The password page comes but once I put my password the page turns white.

**Reply by jsjoeio** [reply]
Hmm nothing looks unusual compared to [mine](https://github.com/jsjoeio/code-server-railway-ipad/blob/main/Dockerfile).

Did you notice this happening after anything specific? i.e. adding a specific extension or adding rclone? 

> The password page comes but once I put my password the page turns white.

Does it show any logs in the browser or Railway?

**Reply by AyaanZaveri** [reply]
It works now!
